pulseaudio-control.bash

```diff
@@ -55,7 +55,7 @@ function getNickname() {
 
     if [ -n "${SINK_NICKNAMES[$sinkName]}" ]; then
```

Comment:
Shouldn't this statement first check `[ -n "$sinkName" ]` as well? And while you're at it, i've noticed that the `if [ -z "$nickname" ]` statement at the end this could be replaced with an `else`, right? Can you add that if you don't mind?

Reply:
It doesn't really matter for this one, since it will just lookup `""`, and not find anything, so  `-n` is satisfied as it is. (Unless someone has specified a nickname for empty string of course... in which case.. works as expected I suppose!)

Reply:
Hm, it turns out you're right. I don't know why - I find bats nothing but painful - but you are. :slightly_smiling_face:

Reply:
Not sure, it's just a good practice to explicitly avoid these kind of edge cases IMO.